### PR TITLE
chore: move docs to docs-v1 to make the dir contents more clear AND add some missing refs to docs-v2 in tests

### LINKED ISSUE #8089: Move docs/ directory to docs-v1 to make the fact that that directory holds the `v1` documentation more obvious
Issue tracks moving docs/ directory to docs-v1 to make the fact that that directory holds the `v1` documentation more obvious

### PR DESCRIPTION
**Pull request by aaron-prindle**
fixes #8089

### CODE REVIEW

**Comment by ericzzzzzzz** [comment]
seems like kokoro job got stuck.. manually cancelled that. will re-run later.

**Comment by aaron-prindle** [comment]
Seems all tests are passing now

File: docs-v2/content/en/docs/references/api-v2/grpc.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ weight: 30
 ******
 WARNING!!!
 
-The file docs/content/en/docs/references/api/grpc.md is generated based on proto/markdown.tmpl,
+The file docs-v1/content/en/docs/references/api/grpc.md is generated based on proto/markdown.tmpl,
```
Comment (ericzzzzzzz) [inline docs-v2/content/en/docs/references/api-v2/grpc.md:10]: the related change under docs-v2 dir should be v2 right?

File: proto/v2/markdown.tmpl [hunk 2]
```diff
@@ -7,7 +7,7 @@ weight: 30
 ******
 WARNING!!!
 
-The file docs/content/en/docs/references/api/grpc.md is generated based on proto/markdown.tmpl,
+The file docs-v1/content/en/docs/references/api/grpc.md is generated based on proto/markdown.tmpl,
```
Comment (ericzzzzzzz) [inline proto/v2/markdown.tmpl:10]: this should be v2 ?

**Review by ericzzzzzzz — APPROVED** [review]

File: docs-v2/content/en/docs/references/api-v2/grpc.md (hunk 1 above)
Comment (aaron-prindle) [reply docs-v2/content/en/docs/references/api-v2/grpc.md]: Thanks, fixed

File: proto/v2/markdown.tmpl (hunk 2 above)
Comment (aaron-prindle) [reply proto/v2/markdown.tmpl]: Thanks! fixed now

**Pull request merged**